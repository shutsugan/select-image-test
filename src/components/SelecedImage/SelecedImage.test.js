import React from "react";
import { render } from "@testing-library/react";
import SelectedImage from ".";

test("generate SelectedImage component snapshot", () => {
  const { container } = render(<SelectedImage image={""} />);

  expect(container).toMatchSnapshot();
});
