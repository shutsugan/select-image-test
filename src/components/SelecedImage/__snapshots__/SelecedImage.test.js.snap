// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate SelectedImage component snapshot 1`] = `
<div>
  <div
    class="selected-image shadow"
  >
    <h2 />
    <img
      alt=""
      src=""
    />
  </div>
</div>
`;
